OJP/OJP_FareSupport.xsd

```diff
@@ -176,7 +170,7 @@
 					<xs:documentation>Name of contact person.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="Email" type="EmailAddressType" minOccurs="0">
+			<xs:element name="EMail" type="EMailAddressType" minOccurs="0">
```

Comment:
Any reason why you prefer EMail with capital M?

Reply:
@skinkie Yes, because for me e-mal is composed of two words (e/electronic and mail).

Reply:
In a serious language yes, but not in English. https://www.merriam-webster.com/dictionary/email

Reply:
Haha, well, but it also seems to be a bit a matter of taste. If more people prefer Email to EMail, I wouldn't oppose changing it.

Reply:
I suggest we go with the webster definition.